pathd/path_cli.c

```diff
@@ -550,7 +550,7 @@ DEFPY_NOSH(
 	"Symbolic Name\n"
 	"Dynamic Path\n")
 {
-	char xpath[XPATH_CANDIDATE_BASELEN];
+	char xpath[XPATH_MAXLEN + XPATH_CANDIDATE_BASELEN];
```

Comment:
Using just XPATH_MAXLEN isn't enough?

Reply:
No - only a larger buffer clears the warning.